packages/peregrine/package.json

```diff
@@ -32,6 +32,7 @@
     "@apollo/react-hooks": "~3.1.2",
     "@babel/runtime": "~7.4.2",
     "apollo-client": "~2.6.4",
+    "graphql-tag": "~2.10.4",
```

Comment:
Yes, I meant to bring this up when I mentioned that this was the first query in `peregrine`. 💯